srv/salt/_modules/calculate.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+
+import salt.client
+
+local_client = salt.client.LocalClient()
+
+def _nearest_power_of_two(target):
+    if target <= 0:
+        raise Exception('Can\'t find nearest power of two of invalid value')
+
+    i = 0
+
+    # Based on the logic presented on http://ceph.com/pgcalc/
+    while True:
+        current = 2 ** i
+        next = 2 ** (i + 1)
```

Comment:
Although `next` is not a keyword, it's part of python's built-ins and shouldn't be used as variable name. `next_` is probably fine.